03_homework/homework.sh

```diff
@@ -7,21 +7,49 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. How would you verify the creation of all 5 directories?
+ls -a
 
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
+touch dir1/file1.txt dir2/file2.txt dir3/file3.txt dir4/file4.txt dir5/file5.txt
```

Comment:
Need write `I love data` into these files

Reply:
Missed that, thank you, Emma!